docker-vertica-v2/Dockerfile

```diff
@@ -121,23 +118,7 @@ ENV VERTICA_STARTUP_LOG_DUPLICATE "/proc/1/fd/1"
 
 COPY ./packages/init.d.functions /etc/rc.d/init.d/functions
 
-# For the init program (process 1), we use s6-overlay. This ensures none of the
-# processes we start ever become zombie's. It will also restart long running
-# processes like cron in case they fail.
-#
-# See https://github.com/just-containers/s6-overlay for instructions on how to
-# setup and configure.
-ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-noarch.tar.xz /tmp
-ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-x86_64.tar.xz /tmp
-COPY s6-rc.d/ /etc/s6-overlay/s6-rc.d/
-
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN set -x \
-  # Untar the init program that was downloaded earlier
-  && tar -C / -Jxpf /tmp/s6-overlay-x86_64.tar.xz \
-  && tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz
-
-ENTRYPOINT [ "/init" ]
+ENTRYPOINT [ "/opt/vertica/bin/node_management_agent" ]
```

Comment:
I set this as default entrypoint. But do we want it to br possible to use it as a monolithic container(it will useful if the image is used outside of k8s)?